applications/nrf5340_audio/unicast_client/main.c

```diff
@@ -76,11 +76,11 @@ static void content_control_msg_sub_thread(void)
 
 		switch (msg.event) {
 		case MEDIA_START:
-			unicast_client_start();
+			unicast_client_start(0);
```

Comment:
Magic value

Reply:
Correct

Reply:
We could create a define for the CIG perhaps, I'm just concerned that will lead people to think that they can change it to be multiple CIGs with ease

Reply:
Add check for cig/big_index for all public functions